src/EditorFeatures/Core/Shared/Extensions/GlyphExtensions.cs

```diff
@@ -26,6 +24,8 @@ public static ImageId GetImageId(this Glyph glyph)
             // VS for mac cannot refer to ImageMoniker
             // so we need to expose ImageId instead of ImageMoniker here
             // and expose ImageMoniker in the EditorFeatures.wpf.dll
+            // The use of constants here is okay because the compiler inlines their values, so no runtime reference is needed.
+            // There are tests in AssemblyReferenceTests.cs to ensure we don't regress that.
```

Comment:
Ditto: full file path? Or any chance the unit tests can do a <cref> to this type or something?

Reply:
>  Or any chance the unit tests can do a to this type or something?

Not sure I understand. The tests verify that the DLL is not referenced at all, so by extension no types can be referenced from the DLL.

Reply:
Ah, markdown ate the tag in there. Edited. Just some way to link this code to the actual test itself.